### PR TITLE
Update examples for config files

### PR DESCRIPTION
**Pull request by dcrowe**
- Updates examples to use config files instead of manual config.
- adjust playwright example to reduce error messages from playwright durign exceptions

### CODE REVIEW

File: playwright/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# How to use
+* Install dependencies `$  pip install -r requirements.txt`
+* Copy configuration file `vrt.json` into directory.
```
Comment (pashidlos) [inline playwright/README.md:3]: could you specify where to get example file?

**Review by pashidlos — APPROVED** [review]

**Comment by pashidlos** [comment]
please, merge after `4.7.0` release

File: playwright/README.md (hunk 1 above)
Comment (dcrowe) [reply playwright/README.md:3]: I was hoping we can have a central doc that specifies the structure and how to put it together. We could then link to that in the examples.

**Review by pashidlos — APPROVED** [review]
nice!

**Pull request merged**